test/support/active_record.rb

```diff
@@ -74,9 +74,11 @@ def deserialize(value)
 
   store :credentials, accessors: [:username, :password_ciphertext], coder: JSON
   store :credentials2, accessors: [:username2, :password2_ciphertext], coder: JSON
+  store :credentials3, accessors: [:username3, :password3_ciphertext], coder: JSON
   has_encrypted :credentials2
   has_encrypted :password
   has_encrypted :password2
+  has_encrypted :password3
```

Comment:
Similar to the above, I added this field, as I noticed that we had added `has_encrypted :passwordX` in our previous changes.